src/lang/en.json

```diff
@@ -102,6 +103,100 @@
   "filter_win": "Result",
   "filter_lane_role": "Lane",
   "filter_patch": "Patch",
+
```

Comment:
I did remove them, but I removed whatever was up top. There are only 1 copy of these. The other files had no merge conflicts.